tests/test_consent.py

```diff
@@ -72,6 +73,40 @@ def test_user_consent(self):
         self.assertEqual(rv.json['consent_agreements'][1]['status'],
                           'suspended')
 
+    def test_consent_order(self):
+        self.shallow_org_tree()
+        org1, org2 = [org for org in Organization.query.filter(
+            Organization.id > 0).limit(2)]
+        old = datetime.now() - relativedelta(years=10)
+        older = old - relativedelta(years=10)
+        oldest = older - relativedelta(years=50)
+
+        audit = Audit(user_id=TEST_USER_ID, subject_id=TEST_USER_ID)
+        uc1 = UserConsent(
+            organization_id=org1.id, user_id=TEST_USER_ID,
+            agreement_url=self.url, audit=audit, acceptance_date=older)
+        uc2 = UserConsent(
+            organization_id=org2.id, user_id=TEST_USER_ID,
+            agreement_url=self.url, audit=audit, acceptance_date=oldest)
+        uc3 = UserConsent(
+            organization_id=0, user_id=TEST_USER_ID,
+            agreement_url=self.url, audit=audit, acceptance_date=old)
+        with SessionScope(db):
+            db.session.add(uc1)
+            db.session.add(uc2)
+            db.session.add(uc3)
+            db.session.commit()
+        self.test_user = db.session.merge(self.test_user)
+        self.login()
+        rv = self.client.get('/api/user/{}/consent'.format(TEST_USER_ID))
```

Comment:
Please use a variable name that doesn't conflict with `pdb` commands if you can :)